.changeset/ten-singers-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effect/platform": minor
```

Comment:
Honestly not sure if this should be minor or patch.

Reply:
patch is fine